### DISCUSSION TITLE
GitHub Copilot for Business  is now generally available 🧑‍✈️

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#47503] [Announcements]
### New Release: GitHub Copilot for Business is now generally available 🧑‍✈️

GitHub Copilot is an AI pair programmer that uses OpenAI Codex to suggest code and entire functions in real-time, right from your editor. Copilot improves developer productivity and happiness, reduces disruptions, and increases the amount of time a developer spends doing satisfying work.

[GitHub Copilot](https://github.com/features/copilot) helps developers code faster, focus on solving bigger problems, stay in the flow longer, and feel more fulfilled with their work.

With GitHub Copilot for Business, you can manage access to GitHub Copilot for organizations within your enterprise and accelerate software development & delivery and improve organization efficiency. 

#### Highlights

- Ability to run Copilot and [Visual Studio IntelliCode](https://visualstudio.microsoft.com/services/intellicode/) side by side
- Improved [Security Vulnerability Filter](https://github.blog/2023-02-14-github-copilot-now-has-a-better-ai-model-and-new-capabilities/)
- VPN Proxy Support
- Simple SaaS Sign-Ups

We look forward to collaborating with you in the community so we can make GitHub Copilot the best choice for you and your team. If you have questions, comments, or feedback, reach out in the [GitHub Copilot community](https://github.com/orgs/community/discussions/categories/copilot). 🚀

### Resources
- [Blog post](https://github.blog/2023-02-14-github-copilot-for-business-is-now-available/) 
- [Changelog](https://github.blog/changelog/2023-02-14-github-copilot-for-business-is-now-widely-available/) 
- [Enable GitHub Copilot](https://docs.github.com/en/enterprise-cloud@latest/copilot/overview-of-github-copilot/about-github-copilot-for-business#setting-up-github-copilot-for-business)
- [FAQs](https://github.com/orgs/community/discussions/47318) and [Features](https://github.com/features/copilot#:~:text=Frequently%20asked%C2%A0questions) 
- [Docs](https://docs.github.com/en/enterprise-cloud@latest/copilot/overview-of-github-copilot/about-github-copilot-for-business)

**Comment by stremovskyy** [comment]
you forgot to say that Copilot blocked for Ukraine users even with paid subscription
This how you "support" our in difficult times?

https://github.com/orgs/community/discussions/47577

**Comment by liashchynskyi** [comment]
Why can't I use Copilot with paid subscription in Ukraine? The heck you doing guys? Why would I need to use additional PAID VPN services to use my Copilot PAID subscription?

**Comment by samurainisa** [comment]


**Reply by AlexanderDudnyk** [reply]
Did they elect a Nazi as president?
Do they pay taxes that finance war, murder and terrorism?
Did they do something to change that?

**Reply by stremovskyy** [reply]
self destructed?

**Reply by AlexanderDudnyk** [reply]
Like all of them soon

**Comment by n-r-w** [comment]
Of course, it's funny when people are willing to pay for a service, and the company does everything to prevent it :)

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
